soh/soh/Enhancements/randomizer/static_data.h

```diff
@@ -48,6 +48,11 @@ class StaticData {
       static std::vector<RandomizerCheck> GetOverworldFishLocations();
       static std::vector<RandomizerCheck> GetOverworldPotLocations();
       static std::vector<RandomizerCheck> GetOverworldFairyLocations();
+      static void RegisterFishLocations();
+      static void RegisterFairyLocations();
+      static void RegisterPotLocations();
+      static void RegisterFreestandingLocations();
```

Comment:
are these being called from anywhere other than the shipinit on boot? if not

```suggestion
```

Reply:
The reason they are here is so they are defined as members of StaticData so they can access locationTable.

Reply:
ah, yeah, slightly more complex use case than most shipinit stuff i've seen but that makes sense